src/test/createTestServices.ts

```diff
@@ -1,9 +1,9 @@
 import { SearchApi } from '@openstax/open-search-client';
 import { createMemoryHistory } from 'history';
-import { intl } from '../app/MessageProvider';
 import config from '../config';
 import { BuyPrintResponse } from '../gateways/createBuyPrintConfigLoader';
 import createHighlightClient from '../gateways/createHighlightClient';
+import createIntl from '../gateways/createIntl';
```

Comment:
gateways are supposed to be proxies to remote or third party services, i'd put `createIntl` in `messages`, probably move `MessageProvider` in there too